corehq/apps/app_manager/tests/test_suite_formats.py

```diff
@@ -75,7 +75,7 @@ def test_case_detail_conditional_enum(self, *args):
         <partial>
           <template>
             <text>
-              <xpath function="if(gender = 'male' and age &lt;= 21, $h{key1_varname}, if(gender = 'female' and age &lt;= 21, $h{key2_varname}, if(gender = 'male' and age &gt; 21, $h{key3_varname}, if(gender = 'female' and age &gt; 21, $h{key4_varname}, ''))))">
+              <xpath function="join(' ', if(gender = 'male' and age &lt;= 21, $h{key1_varname}, if(gender = 'female' and age &lt;= 21, $h{key2_varname}, if(gender = 'male' and age &gt; 21, $h{key3_varname}, if(gender = 'female' and age &gt; 21, $h{key4_varname})">
```

Comment:
So figured out this was due to other enum formats having a separate `_xpath_template()` functions. Thats what caused the malformed xpath. I reverted those formats back to the original behavior for now.

The relevant formats are `conditional-enum` and `enum-image`.

I wasn't sure if we want those formats to have the new behavior.

Reply:
(moving my previous comment into thread)

I don't think the new behavior makes sense for `conditional-enum`, since the keys there are expressions, not values.

`enum-image` I could go either way on. It could make sense conceptually: if you select multiple values, you get multiple images. But projects are also sometimes very deliberate about icon spacing/sizing. If it's extra work to support images, I wouldn't bother.